packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -30,6 +30,7 @@ import {SearchField} from '@react-spectrum/searchfield';
 import {StatusLight} from '@react-spectrum/statuslight';
 import {storiesOf} from '@storybook/react';
 import {Switch} from '@react-spectrum/switch';
+import {TagGroup} from '@react-spectrum/tag';
```

Comment:
In the Form story "isDisabled", it makes the TagGroup `description` greyed out like the rest of the form components. Should this Form prop also disable `allowsRemoving` if that prop is set? Also disable the action buttons?

Reply:
> Should this Form prop also disable allowsRemoving if that prop is set?

I would say that seems a bit too opinionated at this point, since there isn't a disabled state for tags.

> Also disable the action buttons?

I would also say this seems a bit too opinionated.

___

I think maybe we should reconsider if TagGroup should even have Form support, since it isn't necessarily an input on its own.

Reply:
Actually, this is already an issue, the buttons will pick up the isDisabled prop off the context, so they are already disabled, including the show more/show less. Here, I tried it out locally.
![Screenshot 2023-01-24 at 3 09 40 PM](https://user-images.githubusercontent.com/698229/214441443-1dcbc8a5-b4d1-44e0-8834-1e5440e965c0.png)

Reply:
@snowystinger good catch, this has been fixed.